WeatherPortal/WeatherPortal/settings.py

```diff
@@ -85,26 +85,25 @@
     }
 }
 
-
 # Password validation
 # https://docs.djangoproject.com/en/4.0/ref/settings/#auth-password-validators
 
 AUTH_PASSWORD_VALIDATORS = [
-    {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
-    },
+    # off safe passwords for test
+    # {
+    #     'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    # },
+    # {
+    #     'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+    # },
+    # {
+    #     'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
+    # },
+    # {
+    #     'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
+    # },
```

Comment:
Czemu nie chcecie walidatorów?

Reply:
Na koniec projektu włączymy walidatory, teraz poprastu dla prostoty wymyslania nowych hasel dla nowych uzytkownikow zostaly wylaczone

Reply:
No i ogólnie wiadomo, że nie ma co sobie życia utrudniać, ale nie kosztem jakości

Reply:
Dzięki za radę to włącze walidatory i jeszcze raz pushne kod :)